### PR TITLE
added pictures for merge sort

### PR DESCRIPTION
**Pull request by stacyburris**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: javascript/code-challenges/mergeSort/BLOG.md [hunk 1]
```diff
@@ -43,18 +43,18 @@ In your blog article, visually show the output of processing this input array:
...
++ ![mergeSort5](assets/mergeSort5.png)
 
-+ ![image 6](/assets/mergeSort6.png)
++ ![mergeSort6](assets/mergeSort6.png)
 
-+ ![image 7](/assets/mergeSort7.png)
++ ![mergeSort7](assets/mergeSort7.png)
 
-+ ![image 8](/assets/mergeSort8.png)
++ ![mergeSort8](assets/mergeSort8.png)
```
Comment (adrianhuebner) [inline javascript/code-challenges/mergeSort/BLOG.md:60]: Looks really great Stacy! Would just like to see some tests added!
